File: OPENAPI-REST-API/openapi-client/cpp-tiny/lib/Models/Modules.cpp


#include "Modules.h"

using namespace Tiny;

Modules::Modules()
{
}

Modules::Modules(std::string jsonString)
{
	this->fromJson(jsonString);
}

Modules::~Modules()
{

}

void
Modules::fromJson(std::string jsonObj)
{
    bourne::json object = bourne::json::parse(jsonObj);


}

bourne::json
Modules::toJson()
{
    bourne::json object = bourne::json::object();


    return object;

}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/modules.dart
part of swagger.api;

class Modules {
  
  Modules();

  @override
  String toString() {
    return 'Modules[]';
  }

  Modules.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
  }

  Map<String, dynamic> toJson() {
    return {
     };
  }

  static List<Modules> listFromJson(List<dynamic> json) {
    return json == null ? new List<Modules>() : json.map((value) => new Modules.fromJson(value)).toList();
  }

  static Map<String, Modules> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, Modules>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new Modules.fromJson(value));
    }
    return map;
  }
}
